cpp/src/codegen/function_registry.h

```diff
@@ -32,6 +33,8 @@ class FunctionRegistry {
   iterator begin() const;
   iterator end() const;
 
+  static const FuncSignatureVector GetSupportedFunctions() { return function_signatures; }
```

Comment:
Yeah I started with but since it is static and the consumer is static too, there seems to be undefined order of initialization. Ideally this should be exported from the function registry itself for both encapsulation and initialization reasons..